superset/views/api.py

```diff
@@ -40,7 +40,15 @@
 if TYPE_CHECKING:
     from superset.common.query_context_factory import QueryContextFactory
 
-get_time_range_schema = {"type": "string"}
+get_time_range_schema = {
+    "type": ["string", "array"],
```

Comment:
@kgabryje @zhaoyongjie Isn't this a breaking change given that we're changing the type from string to array? If yes, we would need to make it backward compatible to avoid reverting this PR given that we're not in a breaking window anymore.

Reply:
it's not a breaking changing, the `['string', 'array']` means that time_range is able to adapt String and Array type.

Reply:
Oh my bad. I missed the `string` type inside the array. Time to get some coffee ☕ !

Reply:
Perfect! Look forward to a cafe time!